zerver/openapi/zulip.yaml

```diff
@@ -8108,6 +8108,13 @@ paths:
 
                           See [PATCH /settings/display](/api/update-display-settings)
                           for api details on display settings.
+                      timezone_auto_update:
+                        type: boolean
+                        description: |
+                          Present if `update_display_settings` is present in `fetch_event_types`.
+
+                          Whether the user wants the timezone displayed in profile to be updated
```

Comment:
Stick with either `timezone` or `time zone`. The UI shows `time zone`.

Reply:
This description is not really correct right. The timezone is not updated automatically. The browser asks the user whether to update the timezone and updates it only if the user agrees. So writing here that the timezone is updated automatically is kind of misleading.  Similiarly `Realm.timezone_auto_update` is also kind of misleading in that sense.  Would be nice if we can find a better name for this.